pypowerbi/client.py

```diff
@@ -12,10 +13,34 @@
 class PowerBIClient:
     default_resource_url = 'https://analysis.windows.net/powerbi/api'
     default_api_url = 'https://api.powerbi.com'
+    default_authority_url = 'https://login.windows.net/common'
 
     api_version_snippet = 'v1.0'
     api_myorg_snippet = 'myorg'
 
+    @staticmethod
+    def get_client(client_id, username, password, authority_url=None, resource_url=None, api_url=None):
```

Comment:
Good idea with the default get_client method, but I think it should be named get_client_with_username_password so it is cohesive with adal.AuthenticationContext.acquire_token_with_username_password naming. Documentation also would be suitable.

Reply:
Good ideas! Should have added tests and docs in the first place. Will add them and update.